src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.RyuJit.cs

```diff
@@ -419,6 +419,9 @@ private void getReadyToRunDelegateCtorHelper(ref CORINFO_RESOLVED_TOKEN pTargetM
                     // TODO
                     ThrowHelper.ThrowInvalidProgramException();
                 }
+
+                if (!targetMethod.IsRuntimeDeterminedExactMethod)
+                    _compilation.NodeFactory.MetadataManager.NoteOverridingMethod(interfaceMethod, targetMethod);
```

Comment:
This part shouldn't be needed - delegate creation goes through `getCallInfo` below so we would catch it there.

We need to re-resolve here for... reasons... but we already resolved in `getCallInfo` like the comment on line 427 says below.

(The reason is that we shall never give RyuJIT a method in runtime determined form because RyuJIT only understands canonical form.)